### PR TITLE
Update to libfabric-1.13.2

### PR DESCRIPTION
**Pull request by jhh67**
This resolves Cray/chapel-private#2355 and closes Cray/chapel-private#2761.

Signed-off-by: John H. Hartman <jhh67@users.noreply.github.com>

### CODE REVIEW

File: third-party/libfabric/libfabric-src/.appveyor.ps1 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[cmdletbinding()] Param()
```
Comment (ronawho) [inline third-party/libfabric/libfabric-src/.appveyor.ps1:1]: I see a bunch of dotFiles here that I don't see when I download the 1.13.2 tarball. Did you copy the release tarball in or is there any chance you copied a git clone in?

Comment (jhh67) [reply third-party/libfabric/libfabric-src/.appveyor.ps1:1]: TBH had some git issues that I thought I resolved but I guess not. Let me try again to clean things up.

**Comment by jhh67** [comment]
I really hate git.
